code/modules/antagonists/changeling/powers/become_headslug.dm

```diff
@@ -30,6 +31,11 @@
 /datum/action/changeling/headslug/proc/become_headslug(mob/user)
 	var/datum/mind/M = user.mind
 	var/list/organs = user.get_organs_zone("head", 1)
+	if(istype(user.loc, /obj/machinery/gibber) || istype(user.loc, /obj/structure/crematorium))
```

Comment:
Could expand this to be `/obj` - or `if(!isturf(user.loc)` , so you get put out of (locked) lockers

Reply:
will do